sdk/clientcore/tools/annotation-processor/README.md

```diff
@@ -0,0 +1,135 @@
+# Client Core Compile-Time Annotation Processor
+
+The client-core annotation processor for introducing compile-time code generation for libraries based on client core
```

Comment:
Should there be a note here, at least temporarily, that "this project is for experimentation and exploring new ideas that may or may not make it into a supported GA release."